src/v2/view-builder/views/ov/ResendNumberChallengeView.js

```diff
@@ -0,0 +1,16 @@
+import { View } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+
+const ResendNumberChallengeView = View.extend({
+  className: 'okta-form-infobox-warning infobox infobox-warning',
```

Comment:
can we use createCallout instead ?

Reply:
I thoght we dint have callout but I see that it is used in other places.

Reply:
Yeah, we have a mix. We have used a template for warning in some places as well  https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/views/ov/ChallengeOktaVerifyPushView.js#L7-L13

Reply:
Using a template works well here as we are using the new i18n syntax to add the resend link, which doesnt seem to render correctly with `createCallout`

Reply:
wow.. didn't knew we already have similar code. We should create something reusable that can handle such views ? 
like handling warnings, clearing them out, common template ?

We can have more such views in future.

Reply:
Having something re-usable would be great, but it needs some more thought as each of these views have some nuance   that this component needs to be able to handle.